modules/4_nodes/nodes.tf

```diff
@@ -17,6 +17,10 @@
 # limitations under the License.
 #
 ################################################################
+locals {
+  wildcard_dns   = ["nip.io", "xip.io", "sslip.io"]
```

Comment:
Can we also move this to ocp.tf locals and pass it to nodes & install module?

Reply:
It is not referenced by any other modules, so no need to make it global.

Reply:
You are passing it to install module right? So suggestion is to populate the domain in ocp.tf and pass it as domain_name to both modules via variables.

Reply:
No, just passed in domain_name created in node module.